.php-cs-fixer.php

```diff
@@ -1,6 +1,18 @@
 <?php
 
-$config = \TYPO3\CodingStandards\CsFixerConfig::create();
-$config->setParallelConfig(\PhpCsFixer\Runner\Parallel\ParallelConfigFactory::detect());
+use PhpCsFixer\Runner\Parallel\ParallelConfigFactory;
+use TYPO3\CodingStandards\CsFixerConfig;
+
+$config = CsFixerConfig::create();
+// This is required as long as we are on PHPUnit 9.x. It can be removed after the switch to PHPUnit 10.x.
+// @see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/8337
+$config->setRules(
```

Comment:
@oliverklee with this you will have only 1 rule in the configuration.

You have to do something like this:
```php
$rules = $config->getRules();
$rules['php_unit_test_case_static_method_calls'] = ['call_type' => 'self', 'methods' => ['createStub' => 'this']];
$config->setRules($rules);
```

Reply:
Ah, good to know. I'll change that accordingly. Thanks for letting me know! :heart: